source/content/guides/decoupled/drupal-backend-starters/02-create.md

```diff
@@ -19,65 +19,70 @@ integration: [--]
 
-You can install the backend starter kit with Build Tools or with the Dashboard Upstream. Review the information below to select the right option for your setup.
+You can create your backend starter kit project with the Dashboard Upstream, the Terminus Decoupled Kit Plugin, or with Build Tools.
 
-### Use the Dashboard Upstream if:
+Review the following sections to select the right option for your setup.
 
-- You require a simpler setup
+Use the **Dashboard Upstream** if:
 
-- The Pantheon repository is your source of truth
+- You require a simpler setup.
+- The Pantheon repository is your source of truth.
 
-### Use Build Tools if:
+Use the **Terminus Plugin** if:
 
-- Testing is an important part of your workflow
+- You want to create a related backend and frontend codebase in a single process.
+- You prefer to use the CLI.
 
-- You don’t want to manually push changes to your code repo
+Use **Build Tools** if:
 
-## Install with Dashboard Upstream
+- Testing is an important part of your workflow.
+- You don’t want to manually push changes to your code repository.
 
-1. [Go to the workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces), then select the **Sites** page.
+## Create with Dashboard Upstream
 
-1. Click the **Front-End Sites** tab and then click **+Create New Site**.
+1. [Go to the workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces), that the new site should be a part of, then select the **Sites** page.
+
+1. Click **+Create New Site**.
+
+1. Select **Front-End Site** when prompted.
 
 1. Select **Drupal for Front-End** under **Create Backend CMS**.
 
     - You can also follow the [Decoupled Drupal Composer Managed](https://dashboard.pantheon.io/sites/create?upstream_id=c76c0e51-ad85-41d7-b095-a98a75869760) link to create your Front-End Site in the Site Dashboard.
 
-    ![select a starter](../../../../images/decoupled-select-starter-new.png)
+    ![Select a Starter](../../../../images/decoupled-select-starter-new.png)
 
-1. Name your site, select your decoupled environment from the **Choose a Workspace for the Site** drop-down menu, and then click **Continue**. You cannot use a . (period) or _ (underscore) for site and Multidev names.
+1. Name your site, select your decoupled environment from the **Choose a Workspace for the Site** drop-down menu, and then click **Continue**.
 
-1. Confirm your organization selection when prompted in the **Confirm Organization Selection** dialogue box. The Deploying Decoupled Drupal Composer Managed progress indicator displays while your site is spinning up.
+    Site and Multidevs names cannot contain a `.` (period) or `_` (underscore).
 
-1. Click **Visit your Pantheon Dashboard** when the site spin up completes.
+    - To do this step through Terminus, replace `my-new-site`, the site description, and the org ([if available](/terminus/commands/org-list)) in `My Team Name` in the following command:
```

Comment:
I want to get these changes published so I'm leaving this and other Terminus interjections here for now. I think there's probably a better way for us to handle them, whether that's in another section or through the use of the `<Accordion>` expand component, but that can be a future iteration.